registry/application/handlers/service_handlers.py

```diff
@@ -243,14 +240,14 @@ def get_daemon_config_for_current_network(event, context):
 
 @exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger, EXCEPTIONS=EXCEPTIONS)
 def get_service_details_using_org_id_service_id(event, context):
-    logger.info(f"event for get_daemon_config_for_current_network:: {event}")
+    logger.info(f"event: {event}")
     path_parameters = event["queryStringParameters"]
     org_id = path_parameters["org_id"]
     service_id = path_parameters["service_id"]
-    org_uuid, service = ServicePublisherRepository().get_service_for_given_service_id_and_org_id(org_id, service_id)
+    response = ServicePublisherService("", "", "").get_service_for_org_id_and_service_id(org_id, service_id)
```

Comment:
In the constructor ServicePublisherDomainService instance is created. Will that succeed if we give bank values?

Reply:
Nope we only need to give if `get_service_for_org_id_and_service_id` uses it.